tests/behaviour/steps/params.rs

```diff
@@ -348,27 +369,134 @@ pub(crate) struct Value {
-                TypeDBValue::DateTimeTZ(date_time.unwrap().and_local_timezone(tz).unwrap())
+                let (datetime, timezone) = Self::parse_date_time_and_remainder(self.raw_value.as_str());
+
+                if timezone.is_empty() {
+                    TypeDBValue::DateTimeTZ(datetime.and_local_timezone(Tz::default()).unwrap())
+                } else if timezone.starts_with('+') || timezone.starts_with('-') {
+                    // TODO: Temporarily create a TZ for this format as well. It should be a separate DateTimeTZ format later!
+                    let hours: i32 = timezone[1..3].parse().unwrap();
+                    let minutes: i32 = timezone[3..].parse().unwrap();
+                    let total_minutes = hours * 60 + minutes;
+                    let fixed_offset = if &timezone[0..1] == "+" {
+                        FixedOffset::east_opt(total_minutes * 60)
+                    } else {
+                        FixedOffset::west_opt(total_minutes * 60)
+                    };
+                    TypeDBValue::DateTimeTZ(
+                        datetime.and_local_timezone(Self::fixed_offset_to_tz(fixed_offset.unwrap()).unwrap()).unwrap(),
+                    )
+                } else {
+                    TypeDBValue::DateTimeTZ(datetime.and_local_timezone(timezone.parse().unwrap()).unwrap())
+                }
             }
             TypeDBValueType::Duration => TypeDBValue::Duration(self.raw_value.parse().unwrap()),
-            TypeDBValueType::String => TypeDBValue::String(Cow::Owned(self.raw_value)),
+            TypeDBValueType::String => {
+                let value = if self.raw_value.starts_with('"') && self.raw_value.ends_with('"') {
+                    &self.raw_value[1..&self.raw_value.len() - 1]
+                } else {
+                    self.raw_value.as_str()
+                };
+                TypeDBValue::String(Cow::Owned(value.to_string()))
+            }
             TypeDBValueType::Struct(_) => todo!(),
         }
     }
+
+    fn parse_decimal_fraction_part(value: &str) -> u64 {
+        assert!(Self::FRACTIONAL_ZEROES >= value.len());
+        10_u64.pow((Self::FRACTIONAL_ZEROES - value.len() + 1) as u32) * value.trim().parse::<u64>().unwrap()
+    }
+
+    fn parse_date_time_and_remainder(value: &str) -> (NaiveDateTime, &str) {
+        for format in Self::DATETIME_FORMATS {
+            if let Ok((datetime, remainder)) = NaiveDateTime::parse_and_remainder(&value, format) {
+                return (datetime, remainder.trim());
+            }
+        }
+        if let Ok((date, remainder)) = NaiveDate::parse_and_remainder(&value, Self::DATE_FORMAT) {
+            return (date.and_time(NaiveTime::default()), remainder.trim());
+        }
+        panic!(
+            "Cannot parse DateTime: none of the formats {:?} or {:?} fits for {:?}",
+            Self::DATETIME_FORMATS,
+            Self::DATE_FORMAT,
+            value
+        )
+    }
+
+    // TODO: A temporary hack
```

Comment:
Just for the existing tests to pass. We'll change it once we figure out what to do with different formats of `DateTimeTZ`s!

Reply:
But it is a sad tech debt atm.

Reply:
Will create a task